zio/anyio/ztests/zjson-gz.yaml

```diff
@@ -1,15 +0,0 @@
-zed: '*'
```

Comment:
920b67a1754c6eec237e6592ea7361d200e4c79f removed this and `zjson-nonewline.yaml`. Was that intentional?

Reply:
They are the same as other tests and were hard to keep updated modulo auto-decompression etc that is already extensively tested.